src/components/admonition/examples/admonition-composite.tsx

```diff
@@ -0,0 +1,68 @@
+import { Component, h, Prop, State } from '@stencil/core';
+
+/**
+ * Composite
+ *
+ * Show different types
+ */
+@Component({
+    tag: 'limel-example-admonition-composite',
+    shadow: true,
+})
+export class AdmonitionCompositeExample {
+    @Prop()
+    public schema: any;
+
+    @State()
+    private props: any = {
+        title: '',
+        description: 'This is my very nice [type]',
+        type: 'tip',
+    };
+
+    public componentWillLoad() {
+        const properties = { ...this.schema.properties };
+
+        this.schema = {
+            ...this.schema,
+            properties: properties,
+            lime: {
+                layout: {
+                    type: 'grid',
+                },
+            },
+        };
+    }
+
+    public render() {
+        const description = this.props?.description?.replace(
+            '[type]',
+            this.props?.type ?? ''
```

Comment:
Aww, I thought this was part of the component… 😞

But, I guess this wouldn't actually be very useful in normal use, and it would spend runtime resources for no good reason. So let's keep it to the example only 👍